api/_mixins/ParentNode__Element.json

```diff
@@ -724,7 +402,7 @@
       },
       "replaceChildren": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ParentNode/replaceChildren",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/replaceChildren",
```

Comment:
Things to update outside of the diff:
- opera_android should be "61"
- safari and safari_ios should be "14" based on https://trac.webkit.org/changeset/262381/webkit, just like for Document and DocumentFragment